pkg/agent/util/net_windows.go

```diff
@@ -270,34 +281,73 @@ func SetLinkUp(name string) (net.HardwareAddr, int, error) {
 	return mac, index, nil
 }
 
-func ConfigureLinkAddress(idx int, gwIPNet *net.IPNet) error {
-	if gwIPNet.IP.To4() == nil {
-		klog.Warningf("Windows only supports IPv4 addresses. Skip this address %s", gwIPNet.String())
-		return nil
-	}
-
+func ConfigureLinkAddresses(idx int, ipNets []*net.IPNet) error {
```

Comment:
`ConfigureLinkAddresses` is a common function.

I think we need to add comments let caller know that the existed unused addresses will be removed.

Reply:
I am not sure what you mean by common function, but I added a function-level comment for both the Linux and the Windows version